### PR TITLE
Additions to the {targets} chapter... and fixing the rendering of the python chapter

### PR DESCRIPTION
**Pull request by steeleb**
Heya!

Two bits here:

1. I added text to the {targets} chapter! I re-ordered things here (so it looks like I changed every bit of your text, even though I didn't) and added a bit about branching and python scripts. Please do look over them to make sure (in particular) that my {targets} syntax makes sense!
2. I realized that I had the python in R chapter in the wrong format and so it wasn't rendering in the Bookdown. I fixed that and so there's an extra file and some edits to that .Rmd.

WAHOOOOO!

### CODE REVIEW

File: 09-Workflow-Management-with-targets.Rmd [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+upstream code or datasets. This means that you as a human have no need to
+manually re-run scripts that are outdated by changes in code; {targets} knows
+which are affected, can tell you, and automatically re-run them.
+
+Not every analysis is a perfect fit for a {targets} workflow, however. Very
+small analyses may not be worth the time needed to invest to build out the
+structure of the pipeline. Workflows that are often re-run with updated
+datasets or which can perform many parallel iterations are especially good
+candidates for the {targets} pipeline style. Because the ROSSyndicate values
+literate code and {targets} does not work with Markdown or Quarto documents,
```
Comment (mbrousil) [inline 09-Workflow-Management-with-targets.Rmd:31]: Targets actually does work with these! [tarchetypes::tar_render](https://docs.ropensci.org/tarchetypes/reference/tar_render.html) and [tarchetypes::tar_quarto](https://docs.ropensci.org/tarchetypes/reference/tar_quarto.html). Quarto has been buggy for me in the past and bookdown is hard, though possible, to implement.

File: 09-Workflow-Management-with-targets.Rmd [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+there are additional steps to creating easilyt understandable and literate code
```
Comment (mbrousil) [inline 09-Workflow-Management-with-targets.Rmd]: Typo with "easily"

File: 09-Workflow-Management-with-targets.Rmd (hunk 1 above)
Comment (steeleb) [reply 09-Workflow-Management-with-targets.Rmd:31]: Yes, yes. I should clarify that it doesn't render *and* track the output, aka it's a literate/visual output, but not compatible with the 'return' of values/environments, right? I'm trying to get at that aspect of {targets} that you learned about earlier this year.

File: 09-Workflow-Management-with-targets.Rmd (hunk 2 above)
Comment (steeleb) [reply 09-Workflow-Management-with-targets.Rmd]: fixed in 83dec9e

File: 09-Workflow-Management-with-targets.Rmd (hunk 1 above)
Comment (mbrousil) [reply 09-Workflow-Management-with-targets.Rmd:31]: AH yes, basically. That's what I've managed to get {bookdown} to do, but it's sketchy

**Review by mbrousil — APPROVED** [review]

**Pull request merged**